KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyBalanceView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  KarhooLoyaltyBalanceView.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 17.01.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class KarhooLoyaltyBalanceView: UIView {
+    private var balance: Int = 0
+    
+    // MARK: - UI
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.accessibilityIdentifier = KHLoyaltyViewID.balanceLabel
+        label.text = getTitle()
+        label.textAlignment = .center
+        label.font = KarhooUI.fonts.captionBold()
+        label.accessibilityLabel = getTitle()
+        return label
+    }()
+    
+    // MARK: - Init
+    init() {
+        super.init(frame: .zero)
+        self.setupView()
```

Comment:
`self` redundant. We should avoid using `self` without any need - thanks to this approach, every time we see `self` we can be more aware of things like strong reference cycle. Kinda yellow flag :)